params/config.go

```diff
@@ -647,6 +757,33 @@ func (c *ChainConfig) CheckConfigForkOrder() error {
 			lastFork = cur
 		}
 	}
+
+	// Check that all forks with blobs explicitly define the blob schedule
+	// configuration.
+	bsc := c.BlobScheduleConfig
+	if bsc == nil {
+		bsc = &BlobScheduleConfig{}
+	}
+	for _, cur := range []struct {
+		name      string
+		timestamp *uint64
+		config    *BlobConfig
+	}{
+		{name: "cancun", timestamp: c.CancunTime, config: bsc.Cancun},
+		{name: "prague", timestamp: c.PragueTime, config: bsc.Prague},
+		{name: "verkle", timestamp: c.VerkleTime, config: bsc.Verkle},
```

Comment:
our target is to make `master` able to sync the verkle testnet, which is not built on top of cancun/prague. Please remove that line, we can add it back when a new testnet is going to be created.